exercism/diamond/diamond.js

```diff
@@ -0,0 +1,44 @@
+var Diamond = function() {}
+
+Diamond.prototype.makeDiamond = function(letter) {
+  if (letter === 'A') {
+    return 'A\n'
+  }
+  var alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ".split('')
+  var character = ' '
+  var middle = alphabet.indexOf(letter) + 1
+
+  var tempStorage = []
+  for (var i = 0; i < middle; i++) {
+    tempStorage[i] = []
+  }
+
+  for (var i = 1; i < middle; i++) {
+    tempStorage[i].push(alphabet[i])
+  }
+
+  for (var i = 0; i < middle; i++) {
+    var spacesBeforeAndAfter = new Array((middle) - i).join(character)
+    tempStorage[i].push(alphabet[i])
+    tempStorage[i].unshift(spacesBeforeAndAfter)
+    tempStorage[i].push(spacesBeforeAndAfter)
```

Comment:
This is pretty cool! I would have never though to just push on either side! Nice 👍